proto/dwtools/abase/l0.test/Buffer.test.s

```diff
@@ -629,6 +629,11 @@ function constructorIsBuffer( test )
   var expected = true;
   test.identical( got, expected );
 
+  test.case = 'check empty U8x';
```

Comment:
Please, add word `instance`:
`check empty U8x` -> `check empty U8x instance`, 
Please, check other test cases. 

The description of test case should have correct information.